README.md

```diff
@@ -363,6 +363,86 @@ An optional unique identifier for the notification or an identifier for a batch
 </details>
 
 
+### Precompiled Letter
+
+#### Method
+
+The letter must contain:
+
+- A reference for the letter, used to identify the letter (this MUST NOT contain personal data)
+- The file content - a base64 encoded string 
+
+A pre-compiled letter can be sent in two ways: as a Java File object or a base64 string.
+If a File object is provided it is converted to a base64 string by the client.
+
+<details>
+<summary>
+Java File Object - Click here to expand for more information.
+</summary>
+
+```java
+File precompiledPDF = new File("<path to your file>");
+SendLetterResponse response = client.sendPrecompiledLetter("Your reference", precompiledPDF)
+
+```
+</details>
+
+<details>
+<summary>
+Base64 Ecoded String - Click here to expand for more information.
+</summary>
+
+```java
+String base64EncodedPDFFile = "<base64 encoded string of a file>";
+SendLetterResponse response = client.SendLetterResponse sendPrecompiledLetter("Your reference", base64EncodedPDFFile);
+```
+</details>
+
+#### Response
+
+If the request is successful, the SendLetterResponse is returned from the client. Attributes of the SendLetterResponse are listed below.
+
+<details>
+<summary>
+Click here to expand for more information.
+</summary>
+
+```java
+	UUID notificationId;
+	Optional<String> reference;
+	UUID templateId;
+	int templateVersion;
+	String templateUri;
+	String body (for recompiled letters this is always null);
+	String subject;
+```
+
+Otherwise the client will raise a `NotificationClientException`:
+
+|`error.status_code`|`error.message`|
+|:---|:---|
+|`429`|`[{`<br>`"error": "RateLimitError",`<br>`"message": "Exceeded rate limit for key type live of 10 requests per 20 seconds"`<br>`}]`|
```

Comment:
Do we have a file size constraint and error message for that?

Reply:
No file size constraint at the moment